src/main/java/pl/futurecollars/invoicing/utils/FilesService.java

```diff
@@ -0,0 +1,50 @@
+package pl.futurecollars.invoicing.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
+public class FilesService {
+
+  public void createRepository(Path idPath, Path databasePath) throws IOException {
+    createIdPathRepository(idPath);
+    createDatabasePathRepository(databasePath);
+  }
+
+  private void createIdPathRepository(Path idPath) throws IOException {
```

Comment:
Zmieniłbym nazwę tej metody, ponieważ patrząc na jej implementację to ona nic nie wie (i nie powinna) o tym jakiego przeznaczenia będzie dany plik.
Ona ma tylko utworzyć nam plik na podstawie ścieżki, którą dostanie.

Natomiast linijka "writeToFile(idPath, String.valueOf(0L));" to już rzecz, która powinna znajdować się pod wywołaniem metody createRepository (tzn. tam gdzie to robisz). Nie powinna znajdować się w tej klasie, ponieważ jest to klasa do obsługi plików i nie powinna wiedzieć czegokolwiek o logice biznesowej.